src/components/header/components/Navigation.jsx

```diff
@@ -201,9 +204,9 @@ const Navigation = ({ isMenuOpen, setIsMenuOpen, mobile, tablet }) => {
           {!mobile && <CartPicto />}
           {mobile && <p>Cart</p>}
           {/* Picto notification up the cart icon */}
-          {showCart?.length > 0 && (
+          {showCart?.length !== 0 && (
```

Comment:
Just to know why change it it was stuck?

Reply:
Don't know TBH, I could see cart with 0 items in it. 
But I think we could do : 
`showCart?.length && ()`

Reply:
Nice work on this side